.github/workflows/branch_sync.yml

```diff
@@ -0,0 +1,103 @@
+name: Sync PR
+
+on:
+  schedule:
+    - cron: '33 15 * * 1-5'  # 15:33 UTC Mon-Fri
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: Branch to merge into master
+        required: false
+  # push:
+  #   branches:
+  #     - master
+  #     - '8.*.x'
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    env:
+      BRANCH: ${{ inputs.branch || '8.1.x' }}
```

Comment:
Otherwise we could potentially retrieve the 8.*.x branch name(s) from https://github.com/cylc/cylc-admin/blob/master/docs/status/branches.json